<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="PatientsPage">
     <widget class="QPushButton" name="addPatient_pushButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>50</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add a New Patient</string>
      </property>
     </widget>
     <widget class="QPushButton" name="patientsPage_exit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>520</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>160</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Filter Patients:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>761</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AddPatientPage">
     <widget class="QPushButton" name="addPatientCancelButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>530</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>771</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="name_label_4">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="name_line_edit_4"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="dob_label_4">
         <property name="text">
          <string>Date of Birth:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="dob_date_edit_4"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Sex:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Height (inches)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Weight (lbs)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Social Security Number</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Home Address</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Email Address:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Phone Number:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Emergency Contact Name:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Emergency Contact Number:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Emergency Contact Email:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="sex_comboBox">
         <item>
          <property name="text">
           <string>Select</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Male</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Female</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Other</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="height_spinBox"/>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="weight_spinBox">
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="ssn_lineEdit"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="address_lineEdit"/>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="email_lineEdit"/>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="phone_lineEdit"/>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="ec_name_lineEdit"/>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="ec_phone_lineEdit"/>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="ec_email_lineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="addPatientSubmitButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>530</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Patient</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="HomePage">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>321</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>My Calendar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="patientLookupButton">
         <property name="text">
          <string>Patient Lookup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Sign Out</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMentcare">
    <property name="title">
     <string>Mentcare</string>
    </property>
   </widget>
   <addaction name="menuMentcare"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
